builds/fabric8-che/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
-        <version>@che.version@</version>
```

Comment:
Do you have any idea when you will be able to merge #89 ? it seems it depends on the merge of the `openshift-connector-rebased` branch of `eclipse/che`, which might take some more time.

What is the exact problem with this line ? Is it the fact that, on your `openshift-connector-rebased` branch, you will need to change the `5.6.0-openshift-connector-SNAPSHOT` version to make it consistent with the upstream master version (which you already did for the root `pom.xml` [here](https://github.com/redhat-developer/rh-che/pull/89/files#diff-600376dffeb79835ede4a0b285078036R25) ?